openedx/features/survey_report/templates/survey_report/admin_banner.html

```diff
@@ -31,6 +31,7 @@ <h1 style="margin: 0; color: #FFFF; font-weight: 600;">Join the Open edX Data Sh
 {% endif %}
 
     <!-- The original content of the block -->
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
```

Comment:
Should we load jquery using the file in the vendor dir?
Such as:
https://github.com/openedx/edx-platform/blob/ff3d21f186a8acfcc441e74a5edcf661ea89b873/lms/templates/verify_student/test/fake_softwaresecure_response.html#L24

Reply:
Sure thing I changed the jquery accordingly.